data/preprocess/util.py

```diff
@@ -0,0 +1,180 @@
+import numpy as np
+
```

Comment:
This looks good generally. Some methods lack docs (brief suffices) about what they do/their argument.s